prefoil/airfoil.py

```diff
@@ -13,7 +13,7 @@
 from scipy.optimize import brentq, newton, minimize
 from . import sampling
 from .utils.geom_ops import _translateCoords, _rotateCoords, _scaleCoords, _getClosestY
-from .utils.io import _writeDat, _writePlot3D, _writeFFD, Error
+from .utils.io_utils import _writeDat, _writePlot3D, _writeFFD, Error
```

Comment:
Why the name change?

Reply:
If you do really really want to change it, you'll also need to update the __init__.py file in the utils directory